lib/db/mysql/patches/patch-016-017.sql

```diff
@@ -0,0 +1,6 @@
+--  Add `lastUsedAt` column to the `refreshTokens` table.
+
+ALTER TABLE refreshTokens ADD COLUMN lastUsedAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
+ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
⚠️  @larissagaulia the syntax seems wrong here, I see `CURRENT_TIMESTAMP;` I think it should be a comma?